Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -6,6 +6,7 @@
 using LootLocker.LootLockerEnums;
 using System.Linq;
 using System.Security.Cryptography;
+using static UnityEngine.UIElements.UxmlAttributeDescription;
```

Comment:
What is this one being used for? 
Not against it, just can't see where it's being used.

Reply:
Definitely a bad import, will fix